### PR TITLE
Ar fac updates

### PR DESCRIPTION
**Pull request by alexrichey**
This is just a handful of various tweaks/notes made during the facdb data ingestion. (Note that one of the commits is just black'ing the ingestion.py file, so going commit-by-commit is a touch easier)

### CODE REVIEW

**Comment by fvankrieken** [comment]
Sorry if I'm missing the context elsewhere - why the change in process for nysed nonpublicenrollment?

File: library/ingest.py [hunk 1]
```diff
@@ -111,6 +116,16 @@ def wrapper(self, *args, **kwargs) -> list:
                 def update_progress(complete, message, unknown):
                     progress.update(task, completed=floor(complete * 1000))
 
+                # This addresses a gdal issue where translation will fail
+                # to generate a csv if one already exists at the dest path
```
Comment (fvankrieken) [inline library/ingest.py:120]: Does the translation for this case fail even with `"OVERWRITE=YES"` as an option for destination in the template yml?

And if the answer is no, seems like we'd want that always supplied, I can't think of a case where we wouldn't want to overwrite the local destination file

**Comment by alexrichey** [comment]
> Sorry if I'm missing the context elsewhere - why the change in process for nysed nonpublicenrollment?

@fvankrieken Context is [here](https://github.com/NYCPlanning/db-data-library/pull/410/commits/249ec6cf952d5ced44189d735d04f81e4aafa719) in the commit message body

File: library/ingest.py (hunk 1 above)
Comment (fvankrieken) [reply library/ingest.py]: Gotcha, weird. No, this seems fine!

**Comment by fvankrieken** [comment]
> > Sorry if I'm missing the context elsewhere - why the change in process for nysed nonpublicenrollment?
> 
> @fvankrieken Context is [here](https://github.com/NYCPlanning/db-data-library/pull/410/commits/249ec6cf952d5ced44189d735d04f81e4aafa719) in the commit message body

That's `nysdec lands`, I was asking about `nysed nonpublicenrollment`

**Comment by alexrichey** [comment]
> That's `nysdec lands`, I was asking about `nysed nonpublicenrollment`
@fvankrieken 
My mistake! So this one does fail, I believe because the xlsx has two tabs. I should probably just specify the tab in the py script though. I'll give that a shot, and if it doesn't work then I'll add some context in the commit message.

**Comment by alexrichey** [comment]
@fvankrieken Specifying the sheetname works.

Additionally, @fvankrieken / @damonmcc related PR [here](https://github.com/NYCPlanning/data-engineering/pull/76).

**Review by fvankrieken — APPROVED** [review]

**Pull request merged**